content/download/package-manager.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Package Manager
+description: Installing Node.js via package manager
+authors: fhemberger, XhmikosR, shadowspawn, vsemozhetbyt, nschonni, wildcard, MrJithil, kasicka, cassidyjames, Trott, richardlau, Qantas94Heavy, pierreneter, 0mp, ThePrez, PoojaDurgad, MaledongGit, Megajin, marc-maurer, yodeyer, geek, sudowork, strawbrary, ryanmurakami, rbnswartz, arkwright, oliversalzburg, mweagle, Mohamed3on, Ginden, kapouer, jperkin, jericopulvera, jedsmith, jasonkarns, sonicdoe, mcollina, fornwall, danbev, naskapal, awochna, AdamMajer, ahmetanilgur, bnb, qbit
+category: download
+---
+
+# Installing Node.js via package manager
```

Comment:
You might want to removing this part and moving it to the title of the page.

```suggestion

```

Reply:
@lancemccluskey This part needs to be removed, or else it'll get repeated on the page

Reply:
I tried that, but its still failing locally

Reply:
> I tried that, but its still failing locally

I think the tests are failing for some other reasons. I'm asking to remove this line because it'll get repeated twice on the page.

![image](https://user-images.githubusercontent.com/54291836/126066878-2f078701-7628-4908-9659-fc5cdc818523.png)

Reply:
I know, but for some reason its still failing to display the page. Is there a character limit to titles?

Reply:
nvm i got it